tests/index.js

```diff
@@ -15,6 +15,8 @@ import cacache from 'cacache';
 import isGzip from 'is-gzip';
 import zlib from 'zlib';
 
+import removeIllegalCharacterForWindows from '../scripts/removeIllegalCharacterForWindows';
```

Comment:
This should probably be in `src/utils` instead of `scripts/` otherwise it won't get published.

Reply:
@loveky yep, move this to `tests/utils`